examples/ra8875_bmptest.py

```diff
@@ -27,6 +27,9 @@
 display.init()
 display.fill(WHITE)
 
+def color555(self, rgb):
```

Comment:
I think this should be renamed something like `cast_to_565` or something similar. Compared to `color565` the name suggests that it's converting _to_ `555`. 

Alternatively you could import `color565` as `color_565_from_24bpp` and then make your own `color565` that does the appropriate conversion based on bpp? Not a deal breaker either way

Reply:
I just went with convert_555_to_565()